src/altscore/borrower_central/model/accounting_line_items.py

```
@@ -0,0 +1,199 @@
+        description="End of accrual period (yyyy-mm-dd)"
+    )
+    # Parsing confidence
+    confidence_score: Optional[float] = Field(
+        default=None, alias="confidenceScore",
+        description="Confidence score for parsing"
+    )
+    # Created and updated at
+    created_at: str = Field(alias="createdAt")
+    updated_at: Optional[str] = Field(default=None, alias="updatedAt")
+
+    class Config:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)